tox.ini

```diff
@@ -5,8 +5,9 @@ envlist = flake8, django15, django16, django17, django18
 [testenv]
 setenv =
     DJANGO_SETTINGS_MODULE=test_settings
-    PYTHONPATH = {toxinidir}:{toxinidir}/ajax_select
+    PYTHONPATH = {toxinidir}:{toxinidir}/ajax_select:{toxinidir}/tests
```

Comment:
hmm. now there are two sets of tests. they should be together. is there a reason you couldn't add these tests to the others ?